usaspending_api/accounts/views/federal_spending.py

```diff
@@ -23,7 +24,7 @@ def get_queryset(self):
 
         # required query parameters were not provided
         if not (fiscal_year and agency_id):
-            raise ParseError('Missing one or more required query parameters: fiscal_year, agency_id')
+            raise InvalidParameterException('Missing one or more required query parameters: fiscal_year, agency_id')
```

Comment:
Good call to change the error type here--it didn't make sense to raise a 500 when the incoming parameters were wrong.